offchain/danaswap-ui/views/components/head.ejs

```diff
@@ -0,0 +1,22 @@
+<meta charset="utf-8" />
+<meta http-equiv="X-XSS-Protection" content="1;mode=block" />
+<meta http-equiv="X-Content-Type-Options" content="nosniff" />
+<meta http-equiv="Permissions-Policy" content="interest-cohort=()" />
+<meta http-equiv="Referrer-Policy" content="strict-origin-when-cross-origin" />
+<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+<meta name="theme-color" content="#235DF4" />
+<meta name="description" content="" />
+
+<title>DanaSwap App</title>
+
+<link rel="icon" href="./assets/images/favicon.ico" />
+
+<link rel="stylesheet" href="./assets/styles/fontawesome.css" />
+<link rel="stylesheet" id="theme-css" href="./assets/styles/night.css" />
+
+<link
```

Comment:
Pull into the project. Assess how we can a) shave down the size (we aren't using the 1000s of icons it provides) b) we shouldn't use icon fonts even if we use Font Awesome (they *do* offer SVG icons too)

("a" added to post-MVP)

Reply:
I forgot to remove this, now it's gone